examples/voice_send.cr

```diff
@@ -97,6 +95,15 @@ client.on_message_create do |payload|
       end
     end
 
+    # The above code can also equivalently be realised as:
+
+    # Discord.every(20.milliseconds) do
+    #   frame = parser.next_frame(reuse_buffer: true)
+    #   break unless frame
+    #
+    #   voice_client.not_nil!.play_opus(frame)
+    # end
```

Comment:
Look how much cleaner that is. Remove `timed_run` and change the example. Even if you leave `timed_run`, `every` has a much *much* cleaner API and should be used as the default in the examples.